### DISCUSSION TITLE
Temp Request Cont.

### DISCUSSION THREAD

**Discussion by Methematics-LauncherPope** [Guilouz/Creality-Helper-Script-Wiki#546] [Issues]
I posted a few days ago about my K1max not going to the temp that the slicer is requesting. Same deal as before the new update. My printer will follow the temp defaults listed on PRINTER_PARAMS.CONF instead of the slicer setting.

My slicer start gcode is "PRINT_START" and that's it. Is there more I should add to it or is there a setting in the printer config I should change. I've been plagued with this for awhile and although it's not game breaking, it is annoying.

**Comment by Guilouz** [comment]
Are you sure to have the correct start gcode in your slicer ?

<img width="1094" alt="Capture d’écran 2024-05-04 à 23 36 03" src="https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/12702322/7d0cff25-6b2a-47ab-95f1-547e7aa239ce">

**Reply by Methematics-LauncherPope** [reply]
I will copy that start_print you have posted in the photo verbatim and report back. Thank you for the response.

**Reply by Methematics-LauncherPope** [reply]
Alright I followed the START_PRINT you have shown above to the letter and my temps still are not following the slicer request. I'm very puzzled here and cannot find any reasoning as to why it would not be listening. I ask the slicer to request 225 and it stays at 240.

**Reply by philt79** [reply]
Methematics_LauncherPope, did you have any progress on this issue? I am having the same problem with Cura and my Voron 0.1.
